bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/AddCompletion.java

```diff
@@ -0,0 +1,130 @@
+package org.apache.bookkeeper.proto;
+
+import io.netty.util.Recycler;
+import org.apache.bookkeeper.client.BKException;
+import org.apache.bookkeeper.net.BookieId;
+import org.slf4j.MDC;
+
+class AddCompletion extends CompletionValue implements BookkeeperInternalCallbacks.WriteCallback {
```

Comment:
Refactoring by pulling the classes out masks the actual fix here. Can you, for this fix, leave the refactoring aside?